src/test/util_tests.cpp

```diff
@@ -542,80 +542,56 @@ BOOST_AUTO_TEST_CASE(util_GetArg)
     BOOST_CHECK_EQUAL(testArgs.GetArg("pritest4", "default"), "b");
 }
 
-BOOST_AUTO_TEST_CASE(util_GetChainName)
+BOOST_AUTO_TEST_CASE(util_GetChainMode)
 {
     TestArgsManager test_args;
-    const char* avail_args[] = {"-testnet", "-regtest"};
-    test_args.SetupArgs(2, avail_args);
+    const char* avail_args[] = {"-regtest"};
+    test_args.SetupArgs(1, avail_args);
 
     const char* argv_testnet[] = {"cmd", "-testnet"};
```

Comment:
Since `-testnet` parameter was removed, is it correct to not specify it?

Reply:
I want to verify that the parameter "-testnet" does not have any effect on GetChainMode(). This test is trying to check that parsing no argument and parsing -testnet gives the same chain mode, MAIN.

Reply:
I see. it's ok.